tools/jbuf/generate_plot.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+if [ ! -f .env ]; then
+    echo ".env is missing. Copy and edit env-template!"
+    exit 1
+fi
+
+export `cat .env`
+
+echo "target: $target"
+echo "netif:  $netif"
+echo "once:   $once"
+
+if ! which jq; then
+    echo "Install jq"
+    exit 1
+fi
+
+trap "./jitter-off.sh; killall -q baresip" EXIT
+
+function gen_datfile() {
+    ph=$1
+    filename=$2
+
+    jqc='.traceEvents[] | select (.ph == "'"${ph}"'") | .args.line'
+    cat jbuf.json | jq -r "${jqc}" > $filename
```

Comment:
```suggestion
    jq -r "${jqc}" jbuf.json > $filename
```

Reply:
This didn't work.
```
jq: error: Could not open file jbuf.json: Permission denied
```

So I reverted the change.

Reply:
Strange, maybe depends on version, works fine here.